redLayerModule.py

```diff
@@ -368,6 +368,7 @@ def removeSketchesAction(self):
                     self.iface.mapCanvas().scene().removeItem( sketch[3] )
```

Comment:
Anyway this line does not remove the pointed annotation. Could you investigate why?